app/helpers/application_helper/toolbar/x_vm_cloud_center.rb

```diff
@@ -107,6 +107,14 @@ class ApplicationHelper::Toolbar::XVmCloudCenter < ApplicationHelper::Toolbar::B
           N_('Reconfigure this VM'),
           :klass => ApplicationHelper::Button::VmReconfigure
         ),
+        button(
+          :vm_snapshot_add,
+          'pficon pficon-add-circle-o fa-lg',
+          t = N_('Create a new snapshot for this VM'),
```

Comment:
`Create a new snapshot for this Instance`?

Reply:
Copy/paste fail...